### PR TITLE
docs: Get updated readme

### PR DESCRIPTION
**Pull request by IagoLast**

### CODE REVIEW

**Comment by IagoLast** [comment]
@KICKdesigns We moved the PCI-SDK-WEB to its own repo. I've moved the README pull request here.

My feedback:

- It would be nice to have a brief description about the intention of the SDK at the beginning 
- I don't think the requirements section is required. IMO that's obvious 
- Get mobile API key should point to some section at docs.aptopayments.com we can remove this section for now and update the docs asap.
- In the install-sdk section version number is missing an initial "v" --> `v2.0.0`
- In the initialize sdk section:
  - Fix the indentation
- Showing PCI-SDk data
  - The note says the sms will be send always, that's not true, the sms will be send only if the client is not PCI-compliant.
- In the Contributions & Development section
  - Remove the pull-requests link.

**Comment by KICKdesigns** [comment]
@IagoLast  See notes below.

- It would be nice to have a brief description about the intention of the SDK at the beginning. -- **@lukewhyte Could you weigh in on what the intention of the SDK is?**
- I don't think the requirements section is required. IMO that's obvious -- **The requirements section is in all the other READMEs (Android / iOS). If we remove it, it will not be consistent with the other docs. Perhaps @georgedotsmith could weigh in on this?**
- Get mobile API key should point to some section at docs.aptopayments.com we can remove this section for now and update the docs asap. -- This will be removed after the main docs portal is updated. For now, we need to keep this in because there's no page to point to in the docs portal. They're in all the other READMEs as well (Android / iOS).
- In the install-sdk section version number is missing an initial "v" --> v2.0.0 -- **My understanding is that the v was to be left out when using it in the script source. Nonetheless, I've reshuffled that info, to make it more clear.**
- In the initialize sdk section:
 - Fix the indentation -- **Indentation has been fixed**
- Showing PCI-SDk data The note says the sms will be send always, that's not true, the sms will be send only if the client is not PCI-compliant. -- **This has been added for clarification.**
- In the Contributions & Development section Remove the pull-requests link. -- **This has been removed**

**Comment by IagoLast** [comment]
> @IagoLast See notes below.
> 
> * It would be nice to have a brief description about the intention of the SDK at the beginning. -- **@lukewhyte Could you weigh in on what the intention of the SDK is?**

👍 

> * I don't think the requirements section is required. IMO that's obvious -- **The requirements section is in all the other READMEs (Android / iOS). If we remove it, it will not be consistent with the other docs. Perhaps @georgedotsmith could weigh in on this?**
Trust me, for a web sdk those requirements are obviated ;) 

> * Get mobile API key should point to some section at docs.aptopayments.com we can remove this section for now and update the docs asap. -- This will be removed after the main docs portal is updated. For now, we need to keep this in because there's no page to point to in the docs portal. They're in all the other READMEs as well (Android / iOS).

👍 

> * In the install-sdk section version number is missing an initial "v" --> v2.0.0 -- **My understanding is that the v was to be left out when using it in the script source. Nonetheless, I've reshuffled that info, to make it more clear.**


Lol, just discovered both options work :

https://cdn.jsdelivr.net/npm/@apto-payments/pci-sdk-web@v2.0.0/dist/umd/apto-pci-sdk.min.js
https://cdn.jsdelivr.net/npm/@apto-payments/pci-sdk-web@2.0.0/dist/umd/apto-pci-sdk.min.js

> * In the initialize sdk section:
> * Fix the indentation -- **Indentation has been fixed**


👍 

> * Showing PCI-SDk data The note says the sms will be send always, that's not true, the sms will be send only if the client is not PCI-compliant. -- **This has been added for clarification.**

👍 

> * In the Contributions & Development section Remove the pull-requests link. -- **This has been removed**

👍

File: README.md [hunk 1]
```diff
@@ -1,158 +1,170 @@
...
+
+### Get the Mobile API key
+
+A Mobile API Key is required to run the SDK. To retrieve your Mobile API Key:
+
+1. Register for an account or login into the [Apto Developer Portal](https://developer.aptopayments.com). 
+    
+2. Select **Developers** from the menu. Your **Mobile API Key** is listed on this page.
+
+    ![Mobile API Key](readme_images/devPortal_mobileApiKey.jpg)
```
Comment (IagoLast) [inline README.md]:
```suggestion
2. Select **Developers** from the menu. Your **Mobile API Key** is listed on [this page](https://developer.aptopayments.com/#/dashboard/developers/credentials).
```

**Pull request merged**